stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/ChannelListView.kt

```diff
@@ -59,7 +56,28 @@ public class ChannelListView @JvmOverloads constructor(
         }.let(::showToast)
     }
 
-    init {
+    private lateinit var actionDialogStyle: ChannelActionsDialogViewStyle
+
+    public constructor(context: Context) : this(context, null, 0)
+
+    public constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, 0)
+
+    public constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
+        context.createStreamThemeWrapper(),
```

Comment:
Will be already wrapped when accessing `context` from `Fragment` in the init function